Segment-Nielsen-DCR/Classes/SEGNielsenDCRIntegration.m

```diff
@@ -209,7 +209,7 @@ long long returnPlayheadPosition(SEGTrackPayload *payload)
 {
     NSDictionary *adMetadata = @{
         @"assetid" : returnCustomAdAssetId(properties, @"asset_id", settings),
-        @"type" : properties[@"type"] ?: @"",
+        @"type" : properties[@"type"] ?: @"ad",
```

Comment:
is this ternary for sure?

Reply:
@briemcnally Yes, if a customer doesn't pass `property.type`, Nielsen has requested that we fall back to sending "ad" (instead of preroll, midroll, etc.).